rental.rb

```diff
@@ -0,0 +1,19 @@
+# rentak.rb
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+    person.add_rental(self)
+  end
+end
+
+def initialize(date, book, person)
+  @date = date
+  @book = book
+  @person = person
+  person.add_rental(self)
+end
```

Comment:
- [ ] I can see you have two `initialize` methods of the same kind declared in this file. Kindly remove one 👍🏽
- [ ] Also, please allow for the rental adding itself to the @book 👍🏽

- Kindly modify your code to look like this:

```ruby
  def initialize(date, book, person)
    @date = date
    @book = book
    book.add_rental(self)

    @person = person
    person.add_rental(self)
  end
```